### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by GabrielHare**
Adds a missing variable to formatting

### CODE REVIEW

File: CMakeDoxygenFilter.cpp [hunk 1]
```diff
@@ -413,10 +413,10 @@ class CMakeParser
     _os << funcName << '(';
     if (nextToken() == CMakeLexer::TOK_IDENTIFIER)
     {
-      _os << _lexer.getIdentifier();
+      _os << "CMAKE_VARIABLE " << _lexer.getIdentifier();
```
Comment (saschazelzer) [inline CMakeDoxygenFilter.cpp:416]: A similar change was discussed in issue #1 but not included for the stated reasons. Do you have additional input to this discussion?

File: FunctionCMakeDoxygenFilterCompile.cmake [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #!
 #! \brief Download and compile a CMake doxygen input filter
 #!
-#! \param OUT <out-file> (optional) Supply an absolute filename for
```
Comment (saschazelzer) [inline FunctionCMakeDoxygenFilterCompile.cmake:4]: What was the reason for removing this? The `<` and `>` characters should actually be escaped like `&lt;` and `&gt;`.

File: FunctionCMakeDoxygenFilterCompile.cmake [hunk 3]
```diff
@@ -15,7 +15,7 @@
 #! project into a doxygen input filter executable. See
 #! http://github.com/saschazelzer/CMakeDoxygenFilter/blob/master/README for more details.
 #!
-function(FunctionCMakeDoxygenFilterCompile)
+function(FunctionCMakeDoxygenFilterCompile OUT NAMESPACE)
```
Comment (saschazelzer) [inline FunctionCMakeDoxygenFilterCompile.cmake:18]: This looks not right to me, given how the argument parsing below works. What is the intention here?

**Review by saschazelzer — COMMENTED** [review]
Thanks for the PR, please see my comments.

**Comment by GabrielHare** [comment]
You are right on all counts - apologies for the noise.

**Pull request closed without merging**